application/application.bndrun

```diff
@@ -0,0 +1,58 @@
+-include: ../base.bndrun
+
+
+-runrequires.webconsole: \
+	bnd.identity;id='org.apache.felix.webconsole'
+
+-runrequires.flow: \
```

Comment:
Same as above: I don't understand what this is for.

Reply:
in the application l additionally would have the felix webconsole so see state of services/bundles/framework.